bots/happinessncheer.bot.js

```diff
@@ -0,0 +1,48 @@
+const Slack = require('pico-slack');
+const cron = require('node-schedule');
+const {differenceInCalendarDays} = require('date-fns');
+
+const peeps = [
+	`Rbb`,
+	`Christian`,
+	`Evelyn`,
+	`LP`,
+	`Scott`,
+	`Rachel`,
+	`Tina`,
+	`David`,
+	`Katie`,
+	`Meg`,
+	`Greg`,
+	`Kellen`,
+	`Cathleen`,
+	`Chris`,
+	`Simon`,
+	`Carly`,
+	`Jared`,
+	`Mark`,
+	`Jenny`,
+	`Conrad`,
+	`Ross`,
+	`Ryan`,
+];
+
+const PeepOffset = 13; //So whatever date it is we land on the right person
+
+const getSuggester = (offset=0) =>{
+	const now = new Date();
+	const yearStart = new Date(now.getFullYear(), 0, 0);
+	const delta = differenceInCalendarDays(now, yearStart)
+	return peeps[(delta + offset + PeepOffset) % peeps.length];
+}
+
+cron.scheduleJob(`0 22 * * *`, ()=>{
+	const nextUp = getSuggester(1);
+	Slack.send('happiness-and-cheer', `Reminder: ${nextUp} will be picking theme for tomorrow.`);
+});
+
+cron.scheduleJob(`0 9 * * *`, ()=>{
```

Comment:
I'm not exactly sure how `node-schedule` uses timezones, but I presume it will run on server time, which should be UTC, so you will probably need to adjust these schedules. Daylight Savings Time makes things weird, too. There's a `node-schedule-tz` package that might be helpful. https://www.npmjs.com/package/node-schedule-tz

Reply:
Server time is already set to our time ;) so it makes things nice and easy. _Praise be to Heroku, hallowed be thy name_.

Reply:
🛐